### PR TITLE
Project requirements

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).

Create a directory that will contain all the Redux logic (/src/redux)

Configure a Redux store (/src/redux/store.js)

Define a slice of state for books that:

Includes an array of books (initial state: empty array)
Includes a reducer that adds a book
Includes a reducer that removes a book
Define a slice of state for categories that:

Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

### CODE REVIEW

**Comment by Reem-lab** [comment]
Hello @ZabihullahNooriWardak could you check your slack?

File: src/redux/categories/categorySlice.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  {
+    name: 'categories',
+    initialState: {
+      ...initialState,
+      categories: ['under construction'],
+    },
+    reducers: {},
+  },
+);
+
```
Comment (taldr27) [inline src/redux/categories/categorySlice.js:17]: - [ ] Great job on your project! 😄 Just a quick tip, make sure to follow the project requirements by setting your initialState as an empty array. When working with your reducer, you have the opportunity to populate the item with "Under construction." Feel free to check out this example for reference:
   ```
   checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },

   ```

   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @ZabihullahNooriWardak, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional Readme. 
✅ Linters passing.
✅ Correct use of GitHub Flow.
✅ Good job adding redux to your project.
✅ Good job creating your slices.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi @ZabihullahNooriWardak, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.
✅ Good job adding redux to your project.
✅ Good job creating your slices.

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**